src/util/browserPolyfill.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @param {String} localeCode the 'xx-XX' language code for the app
+ * @return {String} the polyfill.io cdn string
+ */
+export function polyfillServiceUrl(localeCode) {
+	const features = [
```

Comment:
Yes, would make a good follow up! I'd like to limit the scope of this and there are more changes coming in.